app/_how-tos/gateway-hybrid-setup.md

```diff
@@ -0,0 +1,211 @@
+---
+title: Run {{site.base_gateway}} in Hybrid Mode
+content_type: how_to
+permalink: /gateway/install/hybrid/
+
+description: Configure separate Control Plane and Data Plane nodes using Hybrid Mode and mTLS
+
+breadcrumbs:
+  - /gateway/
+  - /gateway/install/
+
+products:
+  - gateway
+
+works_on:
+  - on-prem
+tags:
+  - install
+tldr:
+  q: How do I run {{ site.base_gateway }} in Hybrid Mode?
+  a: |
+    Deploy {{ site.base_gateway }} twice, once with `role=control_plane` and once with `role=data_plane`.
+
+    * The `kong.conf` file for the Control Plane must contain connection details for your database
+    * The `kong.conf` file for the Data Plane must contain the Control Plane address, plus the `cluster_cert*` parameters
+
+
+automated_tests: false
+
+prereqs:
+  skip_product: true
+  inline:
+    - title: Install {{site.base_gateway}}
+      content: |
+        {:.warning}
+        > {{ site.base_gateway }} must be installed on both the Control Plane machine <u>and</u> the Data Plane machine before following this how-to
+         
+        To find instructions for your operating system see the [install {{site.base_gateway}}](/gateway/install/#linux) page.
+    - title: Install PostgreSQL
+      content: |
+        {{ site.base_gateway }} requires a database to run in Hybrid mode. For this how-to, please install Postgres on the same node as your Control Plane.
+        
+        To bootstrap your database, follow the [configure a datastore how-to](/how-to/configure-datastore/).
+
+        If you already have a database set up, set the following variables:
+
+        ```bash
+        export KONG_DATABASE=postgres
+        export KONG_PG_HOST=127.0.0.1
+        export KONG_PG_PORT=5432
+        export KONG_PG_USER=kong
+        export KONG_PG_PASSWORD=super_secret
+        export KONG_PG_DATABASE=kong
+        ```
+
+faqs:
+  - q: My Data Plane says "no Route matched with those values"
+    a: |
+      Wait 20 seconds then try again. If it continues to return this message, run `curl localhost:8001/clustering/data-planes` on the Control Plane node and check if the Data Plane is listed.
+---
+
+This how-to explains how to run {{ site.base_gateway }} on-prem in [hybrid mode](/gateway/hybrid-mode/). The quickest way to get started with hybrid mode is with {{ site.konnect_short_name }}.
+
+{% include install/konnect-cta.html%}
+
```

Comment:
```suggestion

If you prefer not to use {{ site.konnect_short_name }}, move on to the next step.
```
Something like this, otherwise the flow is confusing.